LayoutTests/fast/reporting/reporting-observer-lifetime.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html>
+<body>
+<script src="../../resources/js-test-pre.js"></script>
+<script>
+jsTestIsAsync = true;
+if (!window.testRunner || !window.internals) {
+    testFailed("Requires testRunner and internals.");
+    finishJSTest();
+}
+
+let cb = () => { };
+let observer = new ReportingObserver(cb);
+observer.observe();
+
+function runTest()
+{
+    let weakCallback = new WeakRef(cb);
+    let weakObserver = new WeakRef(observer);
+    setTimeout(() => {
+        observer.disconnect();
+        observer = null;
+        cb = null;
+    }, 160);
```

Comment:
I'm just curious; why 160?
>20 means these objects will get marked and be in old rather than eden (I think from the first gc call in the setInterval cb), so I'd guess that >50 would be plenty...

Reply:
Arbitrary number of gc() calls > 2. The entire test run will run in 10 seconds if its a failure and in 160-200ms otherwise.